src/api/emulations/nfs.js

```diff
@@ -1,12 +1,33 @@
 const lib = require('../../native/lib');
 const t = require('../../native/types');
 const nativeH = require('../../native/helpers');
+const consts = require('../../consts');
 
 function isString(arg) {
   return typeof arg === 'string' || (arg.toString ? arg.toString() === '[object String]' : false);
 }
 
 /**
+* Create a new file
+* @returns {File}
+**/
+function newFile() {
```

Comment:
We shouldn't need this at all, but if we do since it's imposed by the safe_client_libs, I think we would only need it within the `NFS.open` function, right here: https://github.com/maidsafe/safe_app_nodejs/pull/104/files#diff-edb2d18591ec6ad75cf8f1c8f46ce0dbR235